test/express.ts

```diff
@@ -117,3 +122,46 @@ Can include **Markdown**.`,
     help: "This is a simple method.\nIt just returns success.",
   });
 });
+
+test("well known services", async (t) => {
+  const app = express();
+
+  const registry = new Registry();
+
+  const service = {
+    hello: (x: { msg: string }) => {
+      console.log("method", x.msg);
+      return `success ${x.msg}`;
+    },
+  };
+  const meta: ServiceDetails<typeof service> = {
+    expose: [{ methodName: "hello" }],
+    service: "hello-service",
+    help: "hello",
+  };
+
+  registry.register(service, meta);
+
+  app.use(
+    "/.well-known/loke-rpc/server",
```

Comment:
May as well show best practices here, use the const `WELL_KNOWN_META_PATH`

Reply:
fixing this now